### PR TITLE
Add DAKEFPVF405X8

### PR DESCRIPTION
**Pull request by ThanYangFPV**
Schematics and configuration files have been emailed to cloudtargets@betaflight.com.

Please review. Thank you.

### CODE REVIEW

File: configs/DAKEFPVF405X8/config.h [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	TIMER_PIN_MAP( 2, PC8 , 2, 1) \
+	TIMER_PIN_MAP( 3, PC9 , 2, 0) \
+	TIMER_PIN_MAP( 4, PA8 , 1, 0) \
+	TIMER_PIN_MAP( 5, PA15, 1, 0) \
+	TIMER_PIN_MAP( 6, PB7 , 1, 0) \
+    TIMER_PIN_MAP( 7, PA1 , 1, 0) \
+    TIMER_PIN_MAP( 8, PA0 , 2, 0) \
+    TIMER_PIN_MAP( 9, PB0 , 2, 0)
+    
+
```
Comment (haslinghuis) [inline configs/DAKEFPVF405X8/config.h:103]:
```suggestion
#define TIMER_PIN_MAPPING \
        TIMER_PIN_MAP( 0, MOTOR1_PIN        , 2,  0) \
        TIMER_PIN_MAP( 1, MOTOR2_PIN        , 2,  1) \
        TIMER_PIN_MAP( 2, MOTOR3_PIN        , 2,  1) \
        TIMER_PIN_MAP( 3, MOTOR4_PIN        , 2,  0) \
        TIMER_PIN_MAP( 4, MOTOR5_PIN        , 1,  0) \
        TIMER_PIN_MAP( 5, MOTOR6_PIN        , 1,  0) \
	TIMER_PIN_MAP( 6, MOTOR7_PIN        , 1,  0) \
	TIMER_PIN_MAP( 7, MOTOR8_PIN        , 1,  0) \
	TIMER_PIN_MAP( 8, LED_STRIP_PN      , 2,  0) \
	TIMER_PIN_MAP( 9, CAMERA_CONTROL_PIN, 2, -1)
```

File: configs/DAKEFPVF405X8/config.h [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_MPU6500
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#define USE_CAMERA_CONTROL
```
Comment (haslinghuis) [inline configs/DAKEFPVF405X8/config.h:45]:
```suggestion
#ifndef USE_CAMERA_CONTROL
#define USE_CAMERA_CONTROL
#endif
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Have not checked timers and DMA

File: configs/DAKEFPVF405X8/config.h [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+#define UART5_TX_PIN         PC12
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PC11
+#define UART5_RX_PIN         PD2
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PC14
+#define LED1_PIN             PC15
```
Comment (ot0tot) [inline configs/DAKEFPVF405X8/config.h:72]:
```suggestion
#define LED0_PIN             PC14
#define LED1_PIN             PC15
```
These pins cannot be used to drive LEDs

**Comment by ot0tot** [comment]
I think we have decided the F405 cannot reliably support 8 motor outputs. Recommend H743 for X8, or reduce the number of motor outputs to use F405.

**Comment by ThanYangFPV** [comment]
> I think we have decided the F405 cannot reliably support 8 motor outputs. Recommend H743 for X8, or reduce the number of motor outputs to use F405.

OK. I will close this PR.
Thank you.

**Pull request closed without merging**